.gitignore

```diff
@@ -4,3 +4,4 @@
 coverage/*
 notes.md
 tests/fixtures/cassette*
+composer.lock
```

Comment:
I totally understand your opinion, I don't know if I was clear enough, I know the users can just update everything and it won't do anything, I mean in a moment someone clones our project to contribute to php-vcr core, the lock file is useful to have a safe state to start patching for us. In that manner do you think it's useful?